projects/1.19/assets/atmospheric/atmospheric/lang/zh_cn.json

```
@@ -0,0 +1,547 @@
+{
+  "advancements.atmospheric.adventure.dunes_pricks.description": "被来自沙丘的每一种带刺植物戳伤",
+  "advancements.atmospheric.adventure.dunes_pricks.title": "四分五裂",
+  "advancements.atmospheric.adventure.loot_arid_garden.description": "在头戴桶形仙人掌的同时，顶着各色针刺登上旱地花园的顶端以夺得宝藏",
+  "advancements.atmospheric.adventure.loot_arid_garden.title": "跑酷大佬！",
+  "advancements.atmospheric.end.forbidden_fruit.description": "你的食物逃跑了...",
+  "advancements.atmospheric.end.forbidden_fruit.title": "禁忌之果",
+  "advancements.atmospheric.husbandry.find_orange.description": "找到长在月桂树上的橙子",
+  "advancements.atmospheric.husbandry.find_orange.title": "有橙子了，你高兴吗？",
+  "advancements.atmospheric.husbandry.finish_gateau.description": "吃下最后一片丝兰果大蛋糕",
+  "advancements.atmospheric.husbandry.finish_gateau.title": "何不食...蛋糕？",
+  "advancements.atmospheric.husbandry.kill_mob_with_passion_fruit.description": "用百香果的种子射杀一个生物",
+  "advancements.atmospheric.husbandry.kill_mob_with_passion_fruit.title": "激情犯罪",
+  "advancements.atmospheric.husbandry.obtain_blood_orange.description": "通过在下界种植月桂树以获得血橙",
+  "advancements.atmospheric.husbandry.obtain_blood_orange.title": "鲁多维科科技",
+  "advancements.atmospheric.husbandry.persistence_while_starving.description": "在最饥饿的时候获得持久效果",
+  "advancements.atmospheric.husbandry.persistence_while_starving.title": "快餐快人",
+  "advancements.atmospheric.husbandry.put_out_fire.description": "在着火时用芦荟叶自救",
+  "advancements.atmospheric.husbandry.put_out_fire.title": "燃烧吧，宝贝！",
+  "advancements.atmospheric.husbandry.spit_passion_fruit.description": "用吃百香果后吐出的百香果种子射击生物",
+  "advancements.atmospheric.husbandry.spit_passion_fruit.title": "噗——",
+  "biome.atmospheric.aspen_parkland": "白杨林地",
+  "biome.atmospheric.dunes": "沙丘",
+  "biome.atmospheric.flourishing_dunes": "繁茂沙丘",
+  "biome.atmospheric.grimwoods": "阴郁树林",
+  "biome.atmospheric.hot_springs": "热泉",
+  "biome.atmospheric.kousa_jungle": "梾树丛林",

```

```suggestion
  "biome.atmospheric.kousa_jungle": "梾木丛林",
```
这里属于我硬抠了，原版的`Birch Forest`译作`桦木森林`，参照修改